index.html

```diff
@@ -5,175 +5,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
-        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
-        crossorigin="anonymous" referrerpolicy="no-referrer" />
-    <title></title>
+    <link rel="stylesheet" href="./css/style.css">
+    <script src="https://kit.fontawesome.com/ebe56293cc.js" crossorigin="anonymous"></script>
+        
+    <title>Abstract Art</title>
 </head>
 
 <body>
     <header>
-        <div class="hamburger"><i class="fa-solid fa-bars"></i></div>
-     
-        
+
         <div class="icons desktop">
-            <div class="black-background-icon">j</div>
-            <div class="gray-background-icon">
-            <ul class="icon-items">
-                <li><span><i class="fa-brands fa-facebook"></i></span></li>
-                <li><span><i class="fa-brands fa-twitter"></i></span></li>
-                <li>English</li>
-                <li>My page</li>
-                <li>Logout</li>
-            </ul> 
-        </div>      
```

Comment:
- [ ] I am seeing that you did not implement the changes requested by the previous reviewer concerning the top navbar having 2 colors instead of a unique color as on the original design. Please fix that.

Reply:
@emmanuellekamwa please look at my nav bar, it only has one color.
![Screenshot from 2022-06-24 12-57-05](https://user-images.githubusercontent.com/100305514/175512095-380ad1be-715f-4a7d-abb6-8230ef041cee.png)